baklava/README.md

```diff
@@ -41,3 +43,11 @@ You can run the app in hot reload mode using `npm run dev`.
 
 While in hot reload mode, any time you save changes to a `ts` file, the electron app will recompile and relaunch itself.
 To exit hot reload mode, use `Ctrl+C` in the console. This will however, leave the current app window open, so be sure to close that also, as it will no longer have access to hot reload.
+
+## Packages needed to build AppImage for linux
+
+- electron-builder
+- electron-builder-notarize
+- builder-utils
+
+These packages should be available globally to build the appimage.
```

Comment:
Not sure if global package requirements is best. Especially since it won't work in the CI/CD

Reply:
So should I remove that line?
I mean for me creating appimage on linux required that so I just wrote it there for someone who wants to create an appimage locally.

Reply:
If you need these dependencies to build the binaries, they should be installed as dev dependencies so that people can just run `npm install` (which is what the gh action uses).

Reply:
It should be fixed now.
I removed that part from the readme and added all the dev dependencies required to build the appimage in `package.json`.